lib/FieldType/DataTransformer/ImageAssetValueTransformer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\RepositoryForms\FieldType\DataTransformer;
+
+use eZ\Publish\Core\FieldType\ImageAsset\Value;
+use Symfony\Component\Form\DataTransformerInterface;
+
+class ImageAssetValueTransformer extends AbstractBinaryBaseTransformer implements DataTransformerInterface
+{
+    public function transform($value)
+    {
+        if (!$value instanceof Value) {
+            return null;
```

Comment:
To make it more future proof I'd rather return `null` for `$value = null` but for `!$value instanceof Value` it should throw `\Symfony\Component\Form\Exception\TransformationFailedException`.